### DISCUSSION TITLE
How to change http headers for dev server?

### DISCUSSION THREAD

**Discussion by nimonian** [vuepress/core#1296] [Q&A]
I need access to SharedArrayBuffer in my project, and so I have to make the site cross-origin isolated.

I have tried this:

```js
// config.js

export default defineUserConfig({
  bundler: viteBundler({
    viteOptions: {
      server: {
        headers: {
          'Cross-Origin-Embedder-Policy': 'require-corp',
          'Cross-Origin-Opener-Policy': 'cross-origin'
        }
      }
    }
  })
})
```

but it is not working.

I did the same with a regular vite/vue project where I had this in the `vite.config.js` instead and it worked no problem.

Is there something I'm not understanding about `viteBundler`? Am I able to modify the http headers in vuepress 2?

**Comment by Mister-Hope** [comment]
Emm, I think this is the correct configuration, @meteorlxy Any ideas?
